control/controlclient/map.go

```diff
@@ -259,10 +262,39 @@ func (ms *mapSession) updateStateFromResponse(resp *tailcfg.MapResponse) {
 		ms.lastDERPMap = dm
 	}
 
+	var packetFilterChanged bool
+	// Older way, one big blog:
 	if pf := resp.PacketFilter; pf != nil {
+		packetFilterChanged = true
+		mak.Set(&ms.namedPacketFilters, "base", views.SliceOf(pf))
+	}
+	// Newer way, named chunks:
+	if m := resp.PacketFilters; m != nil {
```

Comment:
should it be an error to send both?

Reply:
No need? Might even be useful. I'd rather give us implementation flexibility. As long as it's well defined.

Reply:
If something isn't well defined by the docs, let me know.

Reply:
what happens if there is a `PacketFilters["base"]` and `len(PacketFilter) > 0`?